### ISSUE TITLE
Focus indicator missing

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#58344]
### Issue Description:

There is no visible focus outline on this element when it receives focus. 

<img width="361" alt="Image" src="https://github.com/user-attachments/assets/be3b20b3-0c09-4407-bf03-0b195d735f75" />

### Component or Page: 

Component: Filters (v.9.9)

### Name affected element:

Clear Filters button

### Relevant code snippets:

```
element.style {
    outline: none;
    border-width: 1px;
    padding-top: 5px;
    padding-right: 8px;
    padding-bottom: 5px;
    padding-left: 8px;
    text-decoration: none;
}
```

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus.
